react-components/src/hooks/useFdmAssetMappings.tsx

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { type CogniteExternalId } from '@cognite/sdk';
+import { useFdmSdk } from '../components/RevealContainer/SDKProvider';
+import { type Source } from '../utilities/FdmSDK';
+import { type UseQueryResult, useQuery } from '@tanstack/react-query';
+
+export type FdmAssetMappingsConfig = {
```

Comment:
move the `type`'s to new file and let `useFdmAssetMappings.tsx` contain only the hook